controller/control-collection.js

```diff
@@ -0,0 +1,53 @@
+const Note = require('../model/note-model');
+
+const createNote = (note) => {
+    const newNote = new Note({
+        title: note.title,
+        content: note.content,
+        creationDate: Date.now()
+    })
+    console.log(newNote);
+    return newNote.save()
+        .then(
+            () => { return true }
+        )
```

Comment:
For better UX it's better to return the created object so the user don't 
`return await newNote.save()`